CMakeLists.txt

```diff
@@ -298,7 +298,7 @@ else()
         message(FATAL_ERROR "Can not find BLAS or LAPACK!  These are required for Armadillo.  Please install one of them---or install Armadillo---before installing mlpack.")
       endif()
     endif()
-    get_deps(http://files.mlpack.org/armadillo-10.3.0.tar.gz armadillo armadillo-10.3.0.tar.gz)
+    get_deps(https://files.mlpack.org/armadillo-10.3.0.tar.gz armadillo armadillo-10.3.0.tar.gz)
```

Comment:
Suggest to update to a more recent version of Armadillo while we're at it.

At the time of writing, the current version of Armadillo is 12.6.5.

Version 10.3 is quite old;  there have been many bug fixes, improvements and speedups since then.

Reply:
Yeah, good idea; done in 06a2b.